### ISSUE TITLE
Use Heroic's Game Log Feature

### ISSUE THREAD

**Issue by redromnon** [Nocccer/heroic-bash-launcher#1]
Instead of creating a separate log file that Bash Launcher currently does, it'll be useful to use the same log location that Heroic uses after launching games.

i.e. every time the game launches from the bash script, it should be logged to `<appname>-lastPlay.log`. Maybe even add "Running using Bash Launcher" or "Running from launch script" at the top of the contents to show that the game has been launched from the bash script.

**Comment by Nocccer** [comment]
Yes create idea. We should think of which parameter the function needs. Logfile path is a new one for sure

**Comment by redromnon** [comment]
Or maybe just paste the path to the file in here - https://github.com/Heroic-Games-Launcher/heroic-bash-launcher/blob/32441a23bdbe886de27a2c918be0f04b5ed218ba/src/createlaunchfile.ts#L26

Like `exec > <fullpath>heroic/GamesConfig/<appname>-lastPlay.log 2>&1`
